src/main/java/net/minecraftforge/client/model/ItemLayerModel.java

```diff
@@ -429,13 +431,26 @@ private static void putVertex(UnpackedBakedQuad.Builder builder, VertexFormat fo
             case COLOR:
                 builder.put(e, 1f, 1f, 1f, 1f);
                 break;
-            case UV: if(format.getElement(e).getIndex() == 0)
-            {
-                builder.put(e, u, v, 0f, 1f);
+            case UV:
+                if(format.getElement(e).getIndex() == 0)
```

Comment:
Reformatting this muddies the blame, no need for it

Reply:
I do think that the original formatting here makes the scoping less clear, which I feel is partly to blame for the logic error.